google/cloud/spanner_v1/database_sessions_manager.py

```diff
@@ -62,9 +62,10 @@ class DatabaseSessionsManager(object):
     _MAINTENANCE_THREAD_POLLING_INTERVAL = timedelta(minutes=10)
     _MAINTENANCE_THREAD_REFRESH_INTERVAL = timedelta(days=7)
 
-    def __init__(self, database, pool):
+    def __init__(self, database, pool, is_experimental_host: bool = False):
         self._database = database
         self._pool = pool
+        self._is_experimental_host = is_experimental_host
```

Comment:
Please keep the name consistent everywhere `experimental_host` or `_experimental_host`